components/automate-ui/src/app/pages/project/rules/project-rules.component.html

```diff
@@ -14,18 +14,18 @@
         <chef-form-field id="create-name">
           <label>
             <span class="label">Rule Name <span aria-hidden="true">*</span></span>
-            <chef-input ngDefaultControl formControlName="name" (keyup)="handleNameInput($event)"></chef-input>
```

Comment:
Nice that you noticed to remove `ngDefaultControl` since it is no longer being sent to a 3rd party control! However (and technically outside the scope, but since you are in the file), I notice there are several more `ngDefaultControl` in this file that could be removed for the same reason: those were converted from `chef-select` to plain `select`.